queries.sql

```diff
@@ -42,4 +42,26 @@ SELECT COUNT(*) AS animal_never_escape FROM animals WHERE escape_attempts = 0;
 SELECT AVG(weight_kg) AS average_weight FROM animals;
 SELECT neutered, COUNT(*) AS escape_most FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight FROM animals GROUP BY species;
-SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE date_of_birth > '1990-01-01' AND date_of_birth < '2000-01-01' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE date_of_birth > '1990-01-01' AND date_of_birth < '2000-01-01' GROUP BY species;
+
+SELECT name FROM animals JOIN owners ON owners.id = animals.owner_id WHERE owners.full_name = 'Melody Pond';
+SELECT animals.name FROM animals JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
+SELECT owners.full_name AS owner_name, animals.name AS animal_name FROM owners LEFT JOIN animals ON animals.owner_id = owners.id;
+SELECT species.name AS species_name, COUNT(species.name) AS count FROM species JOIN animals ON animals.species_id = species.id GROUP BY species.name;
+
```

Comment:
- [ ] Again here the result of this query should be 4 Pokemon and 6 Digimon. Kindly fix your database so as to return the correct result.

Your result
![image](https://github.com/Esmatullah12/vet-clinic-database/assets/81932478/c2ea799b-f0ad-4b1d-9f34-7666358d5462)

Expected result
![image](https://github.com/Esmatullah12/vet-clinic-database/assets/81932478/f7150ed3-008d-4e25-921f-135cc9646c1a)

Reply:
The reason for the change is the same in this case as well. The animal named 'Agumon' was deleted, and it belonged to the Digimon category. As a result of the deletion, the animal count value is now 5. If 'Agumon' had not been deleted, the animal count would have been 6.
<img width="633" alt="Screenshot 2023-11-04 at 05 39 12" src="https://github.com/Esmatullah12/vet-clinic-database/assets/101590645/695d04a9-d98e-41b9-a43c-c16fb1f7cbf9">

Reply:
Resolved on call, as well I have explained the issue further

Reply:
- SELECT species.name AS species_name, COUNT(species.name) AS count FROM species JOIN animals ON 
<img width="721" alt="Screenshot 2023-11-04 at 14 11 38" src="https://github.com/Esmatullah12/vet-clinic-database/assets/101590645/4927e1ce-818f-4205-9a16-d9f5d1e5ed99">
animals.species_id = species.id GROUP BY species.name;

- SELECT owners.full_name AS owner_name, animals.name AS animal_name FROM owners LEFT JOIN animals ON animals.owner_id = owners.id;
<img width="718" alt="Screenshot 2023-11-04 at 14 07 29" src="https://github.com/Esmatullah12/vet-clinic-database/assets/101590645/cc48fa60-f0e9-4dd0-b641-ae199fbe2200">